### PR TITLE
Complete Shelter Review Page (User Stories 5 & 6)

### PR DESCRIPTION
**Pull request by fieldstyler**
Completed User Stories 5 & 6 that relate to updating a review page.

### CODE REVIEW

File: spec/features/reviews/show_spec.rb [hunk 1]
```diff
@@ -7,12 +7,42 @@
...
+      expect(current_path).to eq("/shelters/#{@shelter1.id}/#{@review.id}/edit")
+
+      fill_in :title, with: "Terrible Service"
+      fill_in :content, with: "We did eight hours just to get my dog!! And when I g"
+      fill_in :rating, with: 5
+
+      click_on "Update Review"
+
+      expect(current_path).to eq("/shelters/#{@shelter1.id}")
+
```
Comment (lkriffell) [inline spec/features/reviews/show_spec.rb:45]: Nice tests. Very robust

File: app/views/reviews/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<div class="form">
+  <%= flash[:alert] %>
+
+<%= form_tag("/shelters/#{@shelter.id}/#{@review.id}", method: :patch) do %>
+  <label>Title:</label><br>
+  <%= text_field_tag :title, @review.title %><br>
+
+  <label>Rating (out of 5): </label><br>
```
Comment (sarynm12) [inline app/views/reviews/edit.html.erb:8]: Thanks for adding the rating scale! Looks good!

**Comment by lkriffell** [comment]
Reviewed by: Logan Riffell

File: spec/features/reviews/show_spec.rb [hunk 3]
```diff
@@ -7,12 +7,42 @@
...
+      fill_in :title, with: "Terrible Service"
+      fill_in :content, with: "We did eight hours just to get my dog!! And when I g"
+      fill_in :rating, with: ""
+
+      click_on "Update Review"
+      expect(page).to have_content("You need to fill in a title, rating, and content in order to update a shelter review")
+      expect(current_path).to eq("/shelters/#{@shelter1.id}/#{@review.id}/edit")
+
+      fill_in :title, with: "Terrible Service"
+      fill_in :content, with: "We did eight hours just to get my dog!! And when I g"
```
Comment (sarynm12) [inline spec/features/reviews/show_spec.rb:39]: Unfinished sentence on 39

**Review by sarynm12 — APPROVED** [review]
This looks great! Added a few comments. We just have one unfinished sentence in our edit and update reviews test. Good work!

File: app/views/reviews/edit.html.erb (hunk 2 above)
Comment (fieldstyler) [reply app/views/reviews/edit.html.erb:8]: Thanks!

File: spec/features/reviews/show_spec.rb (hunk 3 above)
Comment (fieldstyler) [reply spec/features/reviews/show_spec.rb:39]: Oops! Need to fix!

**Pull request merged**